WearComplicationProvidersTestSuite/Wearable/src/main/java/com/example/android/wearable/wear/wearcomplicationproviderstestsuite/Complications.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.android.wearable.wear.wearcomplicationproviderstestsuite
+
+/**
+ * The enumerated list of all complications provided by this app.
+ */
+enum class Complication(
+
+    /**
+     * A stable key that can be used to distinguish between this app's complications.
+     */
+    val key: String
+) {
+    ICON("Icon"),
+    LARGE_IMAGE("LargeImage"),
+    LONG_TEXT("LongText"),
+    RANGED_VALUE("RangedValue"),
+    SHORT_TEXT("ShortText"),
+    SMALL_IMAGE("SmallImage")
+}
```

Comment:
I think it'd make more sense to drop SCREAMING_CASE for PascalCase and use the built-in `name` property of enums in place of the `key` property and the repeated text.

Reply:
I don't have too strong of a preference, but I do like the extra explicitness of a separate `key`.

Reply:
Definitely true, although it's also a bit magical that `name`s of enum cases don't get obfuscated either by Proguard, even though the values themselves are.

On the duplicate note, I'm planning to add tests for these keys, so I will definitely be ensuring that the keys are unique there.